spec/rubocop/cop/infinum/attribute_default_block_value_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
```

Comment:
Can you add for my curiosity a spec with multiline attribute where default: would end up in a new line.

Reply:
Of course :)

<img width="861" alt="Screenshot 2020-08-28 at 14 30 50" src="https://user-images.githubusercontent.com/4703945/91561023-20272500-e93b-11ea-83ec-7c053c70fc0a.png">